ExchangeApp[] installed_apps
ExchangeApp[] available_apps
